src/Model/PhasorDynamics/Exciter/IEEET1/Ieeet1.cpp

```diff
@@ -0,0 +1,355 @@
+        yp_[6] = 0.0;
+        yp_[7] = 0.0;
+        yp_[8] = 0.0;
+
+        return 0;
+      }
+
+      /**
+       * @brief  Identify differential variables.
+       *
+       * @tparam ScalarT Scalar data type
+       * @tparam IdxT Index data type
+       * @return int 0
+       */
+      template <class ScalarT, typename IdxT>
+      int Ieeet1<ScalarT, IdxT>::tagDifferentiable()
+      {
+
+        tag_[0] = true;  // y0 - vts  - Sensed term volt
+        tag_[1] = true;  // y1 - vr   - Votlage reg
+        tag_[2] = true;  // y2 - efdp - Efd pre mult
+        tag_[3] = true;  // y3 - vfx  - Exciter feedback
+        tag_[4] = false; // y4 - vtr  - Term Volt Err
+        tag_[5] = false; // y5 - vf   - Feedback volt
+        tag_[6] = false; // y6 - ve   - Excit. Cntrl Volt
+        tag_[7] = false; // y7 - efd  - Efd
+        tag_[8] = false; // y8 - ksat - Saturation
+
+        return 0;
+      }
+
+      /**
+       * @brief  Scaled sigmoid activation function
+       *
+       * @param[in] x State variable
+       * @tparam ScalarT Scalar data type
+       * @tparam IdxT Index data type
+       * @return Sigmoid approximation evaluated at x
+       *
+       * @warning This needs to be abstracted to be used
+       *          across phasor dynamics. Identical pattern is
+       *          being used in TGOV1 model.
+       *
+       *   Algebraic approximation of transcendental sigmoid.
+       */
+      template <class ScalarT, typename IdxT>
+      ScalarT Ieeet1<ScalarT, IdxT>::sigmoid(ScalarT x)
+      {
+        return ((0.5 * mu_ * x) / (1.0 + std::abs(mu_ * x))) + 0.5;
+      }
+
+      /**
+       * @brief Net Indicator function for regulator limits
+       *
+       * @param[in] x State variable
+       * @param[in] f Conditional derivative of state variable
+       * @tparam ScalarT Scalar data type
+       * @tparam IdxT Index data type
+       * @return Scalar value indicating limit activation.
+       *
+       * @warning This needs to be abstracted to be used
+       *          across phasor dynamics. Identical pattern is
+       *          being used in TGOV1 model.
+       */
+      template <class ScalarT, typename IdxT>
+      ScalarT Ieeet1<ScalarT, IdxT>::indicator(ScalarT x, ScalarT f)
+      {
+
+        ScalarT ind_low  = (this->sigmoid(Vrmin_ - x)) * (this->sigmoid(-f));
+        ScalarT ind_high = (this->sigmoid(x - Vrmax_)) * (this->sigmoid(f));
+        return (1 - ind_low) * (1 - ind_high);
+      }
+
+      /**
+       * @brief Residuals of system equations
+       *
+       */
+      template <class ScalarT, typename IdxT>
+      int Ieeet1<ScalarT, IdxT>::evaluateResidual()
+      {
+
+        // Input Variables
+        ScalarT omega{0};
+        if (speed_signal_)
+        {
+          // Meta PR Note
+          // This seems to be very slow,
+          // but I see how read/write ownership may require this
+          //
+          // I believe implementing the equivilent to signal->read()
```

Comment:
unsure if further typo corrections are really proper to do here but i'll suggest them anyway
```suggestion
          // I believe implementing the equivalent to signal->read()
```